tests/http/test_urls.py

```diff
@@ -0,0 +1,55 @@
+import re
+import uuid
+
+import pytest
+
+from tests.controllers.controller_with_path_parameters import OneTwoEnum
```

Comment:
Please do not import it to avoid cross-tests dependencies without any reason. Just declare required enum in this file.